components/console/helpers/table.rst

```diff
@@ -305,7 +305,7 @@ This results in:
         $table->setHeaders([
             [new TableCell('Main table title', ['colspan' => 3])],
             ['ISBN', 'Title', 'Author'],
-        ])
+        ]);
```

Comment:
This one seems wrong ... the `// ...` implies that we add some rows ... which are later displayed in the example below.

Reply:
Yeah, I assume the user will write something like: 

```php
$table->addRow('...');
```

Which is fine to add a semi colon.